utils/rss-feed.ts

```diff
@@ -0,0 +1,32 @@
+import Parser from 'rss-parser';
+import dotenv from 'dotenv';
+import express from 'express';
+import publicationsJSON from '../publications.json';
+
+const PORT = 5000;
+
+// load the environment variables from the .env file
+dotenv.config({
+    path: '.env',
+});
+
+const publicationsDB = publicationsJSON.publications;
+const FEED_LIST = publicationsDB.map((publication) => {
+    return publication.feed;
+  });
+
+const app = express();
+
+app.get('/', (req, res) => {
+    const parser = new Parser();
+
+    const feedRequests = FEED_LIST.map((feed) => {
+        return parser.parseURL(feed);
```

Comment:
shouldn't we be awaiting here? unsure how `feedRequests` are promises

Reply:
`parseURL` returns a `Promise<Parser.Output>` which is then put into the `feedRequests`.  Also, if we use `Promise.all()` doesn't the input have to be an array of promises? So if we make this await, wouldn't it resolve to `Parser.Output`?